### DISCUSSION TITLE
help enabling linux

### DISCUSSION THREAD

**Discussion by ugh2o** [3kh0/ext-remover#869] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
yeah thats rly it im on v114

**Comment by Aka-but-nice** [comment, marked as answer]
Sure, I'll give you an exploit to enable Linux! I just have a mind-blowing exploit sitting around waiting for a skid to ask about 💀 💀 .

**Reply by ugh2o** [reply]
ironic or....

**Comment by funkymokey** [comment]
You could try chrome://crostini-installer, but I highly doubt it will work.

**Reply by Aka-but-nice** [reply]
nah lil bro why would that work?? it literally goes by the device policy

**Reply by Aka-but-nice** [reply]
if you can't enable crostini from settings chrome://crostini-installer is useless

**Reply by Sxri** [reply]
he said he highly doubts for a reason, no need to call him "lil bro", that just made your reply corny too. 

yeah it goes by device policy, but he still suggested an option, because who knows what ugh2o's skid mind tried.

**Reply by ugh2o** [reply]
my mind is an enigma

**Comment by Gxnnar** [comment]
are there any more urls to install linux,and also it says error instaling linux

**Reply by Aka-but-nice** [reply]
> and also it says error instaling linux

That's... intended? For school accounts with Linux disabled, that link is obviously not going to work. I just gave a suggestion which was all I had at the time.
